app/services/adviser/application_form_validations.rb

```diff
@@ -0,0 +1,86 @@
+class Adviser::ApplicationFormValidations
+  include ActiveModel::Model
+
+  APPLICABLE_DOMESTIC_DEGREE_GRADES = [
+    'First-class honours',
+    'Upper second-class honours (2:1)',
+    'Lower second-class honours (2:2)',
+  ].freeze
+  APPLICABLE_DOMESTIC_DEGREE_LEVELS = %w[bachelor master doctor].freeze
+  APPLICABLE_INTERNATIONAL_DEGREE_LEVELS = %w[
+    bachelor_honours_degree
+    postgraduate_certificate_or_diploma
+    masters_degree
+    doctor_of_philosophy
+    post_doctoral_award
+  ].freeze
```

Comment:
There's a few instances of these being defined throughout the codebase, sometimes inconsistently.  How can we best check that these are always correct (e.g. `First-class` vs `First class`) and are we able to consolidate them at all?

Reply:
As mentioned in Slack, it looks like this is a wider challenge in the codebase and would require some fundamental changes to be able to DRY up the constants.

Reply:
Thanks @ethax-ross. Agree the DRYing should happen separately, but would be good to check that `first-class` vs `first class` etc is correct.

Reply:
Looking through the codebase it appears 'First class' was used historically and newer applications should all be 'First-class'. Given we're only surfacing the adviser service for applications currently being filled out I think we're OK with this as-is?